check-for-missing-dlls.sh

```diff
@@ -67,3 +72,19 @@ do
 	done
 done
 printf "$next_line" >&2
+
+used_dlls_regex="/\\($(sort <"$used_dlls_file" |
+	uniq |
+	sed -e 's/+x/\\+/g' -e 's/\.dll$/\\|/' -e '$s/\\|//' |
+	tr -d '\n')\\)\\.dll\$"
+echo "$all_files" |
+	grep '\.dll$' |
+	grep -v \
+		-e "$used_dlls_regex" \
+		-e '^usr/lib/perl5/' \
+		-e '^usr/lib/gawk/' \
+		-e '^usr/lib/openssl/engines' \
+		-e '^usr/lib/sasl2/' \
+		-e '^mingw../libexec/git-core/\(atlassian\|azuredevops\|bitbucket\|github\|microsoft\|newtonsoft\)\.' \
+		-e '^mingw../lib/\(engines\|reg\|thread\)' |
+	sed 's/^/unused dll: /' >&2
```

Comment:
GH is complaining about no EOL on the last line.

Reply:
That's because it is not needed, and I edited this file in VS Code (which makes it much easier to drop a trailing newline than, say, `vi`).